src/storage/datacore/restapi/mode/alertscount.pm

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright 2024 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+package storage::datacore::restapi::mode::alertscount;
+use strict;
+use warnings;
+use centreon::plugins::misc qw(empty);
+
+use base qw(centreon::plugins::templates::counter);
+
+my %alerts_level = ('trace' => 0, 'info' => 1, 'warning' => 2, 'error' => 3);
+
+sub new {
+    my ($class, %options) = @_;
+
+    my $self = $class->SUPER::new(package => __PACKAGE__, %options, force_new_perfdata => 1);
+    bless $self, $class;
+    $options{options}->add_options(arguments => {
+        'filter-server:s' => { name => 'filter_server' },
+        'max-alert-age:s' => { name => 'max_alert_age' } });
+
+    $self->{output} = $options{output};
+
+    return $self;
+}
+
+sub check_options {
+    my ($self, %options) = @_;
+    $self->SUPER::check_options(%options);
+}
+
+sub set_counters {
```

Comment:
_set_counters_ is often the first function in a mode after the several cutsom output function._